src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -125,6 +126,28 @@ private CallbackResponse handleDefendAllClaim(CallbackParams callbackParams) {
             .build();
     }
 
+    //TODO Duplication Check for the methods to see if it can have a generic ways in the future
+    private CallbackResponse handleAdmitPartOfClaim(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        List<String> errors = paymentDateValidator.validate(Optional.ofNullable(caseData.getRespondToAdmittedClaim())
```

Comment:
Parts of the method look exactly similar to the defendAllClaim methos, can we do something to extract common code to separate place.